spec/models/spam_report_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe SpamReport do
+  let(:first_spam) { create(:work, spam: true, posted: true) }
+  let(:second_spam) { create(:work, spam: true, posted: true, authors: first_spam.authors) }
+  let(:third_spam) { create(:work, spam: true, posted: true, authors: second_spam.authors) }
+
+  before(:example) do
+    ArchiveConfig.SPAM_THRESHOLD = 10
+    create(:work, spam: false, posted: true)
+  end
+
+  it 'has a recent date after the new date' do
+    spam_report = SpamReport.new
+    expect(spam_report.new_date).to be > spam_report.recent_date
+  end
+
+  it 'sends no email if there is no significant spam' do
```

Comment:
Maybe `it "does not send email if the spam score is lower than the spam threshold"`?